flags.go

```diff
@@ -1,21 +1,17 @@
 package main
 
 import (
-	"os"
-	"os/user"
-	"path"
-
 	"github.com/codegangsta/cli"
-
-	log "github.com/Sirupsen/logrus"
+	"os"
+	"path/filepath"
+	"runtime"
 )
 
 func homepath(p string) string {
-	usr, err := user.Current()
-	if err != nil {
-		log.Fatal(err)
+	if runtime.GOOS == "windows" {
```

Comment:
Nevermind, this PR got already merged so I opened https://github.com/docker/swarm/pull/278

Thanks!